src/login/logind-inhibit.c

```diff
@@ -474,6 +474,8 @@ const char *inhibit_what_to_string(InhibitWhat w) {
                 p = stpcpy(p, "handle-hibernate-key:");
         if (w & INHIBIT_HANDLE_LID_SWITCH)
                 p = stpcpy(p, "handle-lid-switch:");
+        if (w & INHIBIT_HANDLE_REBOOT_KEY)
+                p = stpcpy(p, "handle-reboot-key:");
```

Comment:
The buffer still needs to be resized. i.e.

```
static thread_local char buffer[STRLEN(
        "shutdown:"
        "sleep:"
        "idle:"
        "handle-power-key:"
        "handle-suspend-key:"
        "handle-hibernate-key:"
        "handle-lid-switch:"
        "handle-reboot-key")+1];
```

Reply:
Thanks, I completely missed this.
Pushed a fix to it.